tests/scripts/install.sh

```diff
@@ -49,8 +49,8 @@ else
   if [ $REPO_NAME == "trustyai-explainability" ]; then
     echo "Setting manifests in kfctl_openshift to use pull number: $PULL_NUMBER"
     sed -i "s#uri: https://github.com/trustyai-explainability/trustyai-explainability/tarball/main#uri: https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/tarball/pull/${PULL_NUMBER}/head#" ./${KFDEF_FILENAME}
```

Comment:
Minor comment (I think it is not very common) but a fork might have a different name so `${REPO_OWNER}/${REPO_NAME}` can be wrong